web/yo/app/scripts/controllers/gene.js

```diff
@@ -2514,17 +2514,78 @@ angular.module('oncokbApp')
                 mutationEffect.addOn.setText('');
             };
 
-            $scope.move = function(driveList, index, moveIndex, event) {
+            $scope.displayAngle = function (parent, item, type) {
```

Comment:
Let's call this function `displayMoveIcon` even though the font-awesome calls them as angle icons